bluemira/plasma_physics/scaling_laws.py

```diff
@@ -0,0 +1,255 @@
+
+    Notes
+    -----
+    [4] Eich et al., 2011
+        <https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.107.215001>
+    """  # noqa: W505
+    law = PowerLawScaling(
+        constant=0.73e-3,
+        constant_err=0.38e-3,
+        exponents=[-0.78, 1.2, 0.1, 0.02],
+        exp_errs=[0.25, 0.27, 0.11, 0.20],
+    )
+    p_sol /= 1e6
+    value = law(B_t, q_cyl, p_sol, R_0)
+    if error:
+        min_value, max_value = law.calculate_range(B_t, q_cyl, p_sol, R_0)
+        return value, min_value, max_value
+    else:
+        return value
+
+
+def P_LH(n_e, B_t, A, R_0, error=False):  # noqa: N802
+    """
+    Power requirement for accessing H-mode, Martin scaling [3]
+
+    Parameters
+    ----------
+    n_e: float
+        Electron density [1/m^3]
+    B_t: float
+        Toroidal field at the major radius [T]
+    A: float
+        Plasma aspect ratio
+    R_0: float
+        Plasma major radius [m]
+
+    Returns
+    -------
+    P_LH: float
+        Power required to access H-mode [W]
+
+    Notes
+    -----
+    [3] Martin et al., 2008,
+    <https://infoscience.epfl.ch/record/135655/files/1742-6596_123_1_012033.pdf>
+    equation (3)
+
+    \t:math:`P_{LH}=2.15e^{\\pm 0.107}n_{e20}^{0.782 \\pm 0.037}`
+    \t:math:`B_{T}^{0.772 \\pm 0.031}a^{0.975 \\pm 0.08}R_{0}^{0.999 \\pm 0.101}`
+    """  # noqa :W505
+    law = PowerLawScaling(
+        constant=2.15e6,
+        constant_err=0.0,
+        exponents=[0, 0.782, 0.772, 0.975, 0.999],
+        exp_errs=[0.107, 0.037, 0.031, 0.08, 0.101],
+    )
+    leading_term = np.exp(1)
+    n_e20 = n_e / 1e20
+    a = R_0 / A
+    value = law(leading_term, n_e20, B_t, a, R_0)
+
+    if error:
+        min_value, max_value = law.calculate_range(leading_term, n_e20, B_t, a, R_0)
+        return value, min_value, max_value
+    else:
+        return value
+
+
+def IPB98y2(I_p, B_t, p_sep, n, mass, R_0, A, kappa):  # noqa: N802
+    """
+    ITER IPB98(y, 2) Confinement time scaling for ELMy H-mode [2]
+
+    Parameters
+    ----------
+    I_p: float
+        Plasma current [A]
+    B_t: float
+        Toroidal field at R_0 [T]
+    p_sep: float
+        Separatrix power [W]  (a.k.a. loss power (corrected for charge exchange and
+        orbit losses))
+    n: float
+        Line average plasma density [1/m^3]
+    R_0: float
+        Major radius [m]
+    mass: float
+        Average ion mass [a.m.u.]
+    A: float
+        Aspect ratio
+    kappa: float
+        Plasma elongation
+
+    Returns
+    -------
+    tau_E: float
+        The energy confinement time [s]
+
+    Notes
+    -----
+    [2] ITER Physics Expert Group, Nucl. Fus. 39, 12, <https://iopscience.iop.org/article/10.1088/0029-5515/39/12/302/pdf>
+    equation (20)
+
+    \t:math:`\\tau_{E}=0.0562I_p^{0.93}B_t^{0.15}P_{sep}^{-0.69}n^{0.41}M^{0.19}R_0^{1.97}A^{-0.57}\\kappa^{0.78}`
+    """  # noqa :W505
+    I_p = I_p * 1e6  # [MA]
+    p_sep = p_sep * 1e6  # [MW]
+    n = n * 1e19  # [10^19/m^3]
```

Comment:
same here actually, I think this is the wrong way round should probably do this:
```suggestion
    n = raw_uc(n, '1/m^3', '1e19/m^3')  # [10^19/m^3]
```